recipes/sfml/bld.bat

```diff
@@ -0,0 +1,20 @@
+cmake -G "NMake Makefiles" ^
+  -D CMAKE_BUILD_TYPE=Release ^
+  -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+  -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
```

Comment:
I'm wondering where the Windows equivalent of https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/50d5eca739d954e69798f0c4b3b56319a7922236/recipe/activate-gcc.sh#L98 is but this should also not be needed.